nixos/modules/services/misc/ntfy-sh.nix

```diff
@@ -0,0 +1,54 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.ntfy-sh;
+
+  settingsFormat = pkgs.formats.yaml { };
+in
+
+{
+  options.services.ntfy-sh = {
+    enable = mkEnableOption "[ntfy-sh](https://ntfy.sh), a push notification service";
+
+    settings = mkOption {
+      type = types.submodule {
+
+        freeformType = settingsFormat.type;
+
+        options.base-url = mkOption {
+          type = types.str;
+          default = "";
```

Comment:
Is this setting necessary? If so, why provide a default for it? Let it fail so the user can fix it